enhancements/monitoring/hypershift-monitoring.md

```diff
@@ -0,0 +1,392 @@
+
+* Metrics Centralization
+* Extensibility (e.g. one can choose different Observability Products or switch them on demand).
+* Separation of concerns which should result in:
+  * Higher velocity of the Observability Product
+  * Higher velocity of OSD as the offering has to care less about in-cluster observability flows.
+  * Allows managing two products by separate teams without conflicts if one chooses to.
+
+> Note that this does not imply "Centralized Observability Product" to run in a separate cluster and be managed by a separate team. We can mix and match as needed, but strictly for Managed HyperShift use cases we would go for RHOBS service fully managed by Observability Group with AppSRE (SD).
+
+Reasons:
+
+* We can have tighter control on observability budget and spending
+* It’s our common, Red Hat responsibility to offer managed OpenShift. If Observability Group + AppSRE will not provide their support and time, the same (or more) effort and money has to be spent on OSD side (or vendor)
+* The monitoring Team already manages this service.
+  * We already work closely together with AppSRE, tightly integrated with them and familiar with AppInterface.
+  * We have devs on-call and SRE mindset.
+
+## Design Details
+
+We don’t want anyone to wait years to achieve this. As the Observability group, we want to give immediate value. Let’s see how we all can look when deployed. We proposed exact solutions that will have the required functionality, the quickest readiness and optimal stability and efficiency.
+
+In details it looks as below:
+
+![h](assets/hypershift-details.png)
+
+Let’s take a look at all parts:
+
+1. In terms of "Observability Product", we already have a growing open-source [Observatorium](http://observatorium.io/) project that packages and offers opinionated deployments of popular open-source projects for observability needs (e.g Thanos and Loki). What matters is that the Observability Group has a strong impact, deep knowledge and understanding of those projects. We also have ongoing production experience in running and scaling them.
+
+Anyone can install Observatorium, and deploy this service on-premise, but Observability Group also provides [RHOBS](https://rhobs-handbook.netlify.app/projects/observability/rhobs/) as a managed instance of observatorium, that is currently only available to specific internal teams.
+
+Observability Product will be a central pane of view of observability data for SRE needs from multiple clusters. Long term, it will also run critical Prometheus Alerting rules and alert routing for notification purposes.
+
+2. Since the Management cluster is like any other OCP/OSD, we can leverage Platform Monitoring. In near future, we plan to allow [MonitoringStack](https://github.com/openshift/enhancements/pull/866) project to allow scraping platform related metrics, but for the time being, this is done via Platform Monitoring that can remote write relevant metrics to Observability Product.
```

Comment:
Can we generalize this a bit, something like 

>Metrics are scraped by a component running in the management cluster that uses Prometheus remote_write to write them into the observability product. Initially, this will be done by the Platform Monitoring stack but this might change in the future.